server/channels/app/plugin_install.go

```diff
@@ -174,6 +174,16 @@ func (ch *Channels) installPlugin(pluginFile, signature io.ReadSeeker, installat
 		mlog.Warn("Failed to notify plugin status changed", mlog.Err(err))
 	}
 
+	ch.cfgSvc.UpdateConfig(func(cfg *model.Config) {
+		if _, ok := cfg.PluginSettings.Plugins[manifest.Id]; !ok {
+			cfg.PluginSettings.Plugins[manifest.Id] = make(map[string]any)
+		}
+
+		for _, pluginSetting := range manifest.SettingsSchema.Settings {
```

Comment:
should this loop be completely inside the previous if? I've tested this code (once code has strings.toLower) and seem to replace existing config with default one when updating a plugin.

Steps to reproduce:
- install a plugin locally with `make deploy`
- change some values in webapp > system console (UI)
- do another `make deploy` and check config values